frontends/open-discussions/babel.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = api => {
     plugins: [
       "@babel/plugin-transform-flow-strip-types",
       ...(isProduction ? [] : ["react-hot-loader/babel"]),
-      "@babel/plugin-proposal-object-rest-spread",
+      "@babel/plugin-transform-object-rest-spread",
```

Comment:
This package was renamed; it's not a proposal anymore. It was formally accepted (a while ago). The new versions of babel seem to use the new package name.